core-java-modules/core-java-uuid/src/main/java/com/baeldung/uuid/UUIDLongGenerator.java

```diff
@@ -0,0 +1,58 @@
+package com.baeldung.uuid;
+
+import java.nio.ByteBuffer;
+import java.util.UUID;
+
+public class UUIDLongGenerator {
```

Comment:
If the reflection approach stays, let's make it clear that these methods are used:

```suggestion
/**
 * Methods are called by reflection in the unit test
 */
@SuppressWarnings("unused")
public class UUIDLongGenerator {
```